packages/@glimmer/blueprint/files/tests/index.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Glimmer PDP Viewer Tests</title>
```

Comment:
Seems incorrect, probably should be `<%= name %>`

Reply:
Haha, quite right.